english/00_8_Agent/The_name_is_Bond_Milton_Bond.md

```diff
@@ -1,3 +1,6 @@
+You have entered into the world of a secret agent.
+If you used to be a Drill Master, you should aspire to become a [White Tiger](https://play.google.com/store/apps/details?id=com.linecorp.LGRGS) instead.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.